### PR TITLE
fix(ui): fix missing discussion link when preview is unavailable

### PR DESCRIPTION
**Pull request by wa0x6e**
### Summary

When discussion link preview is unavailable, nothing is shown.
It should default to show the bare link

Closes: #86

### How to test

Before the fix:

1. Go to a proposal with an invalid discussion link (e.g. https://testnet.snapshotx.xyz/#/s-tn:wan-test.eth/proposal/0x04a142d72b0d9b004965be01c3eff9bdad7888ceccf7e053cde129ba92a4ab03)
2. It should not be able to show the discussion preview and link, and just the title like 

Screenshot 2024-02-26 at 23 14 14

After the fix:

1. Go to a proposal with an invalid discussion link (e.g. https://testnet.snapshotx.xyz/#/s-tn:wan-test.eth/proposal/0x04a142d72b0d9b004965be01c3eff9bdad7888ceccf7e053cde129ba92a4ab03)
2. It should show the bare discussion link, without the preview like:

Screenshot 2024-02-26 at 23 11 45

In proposal Editor, it should show the preview only when the link is valid, and show nothing otherwise

### CODE REVIEW

**Comment by bonustrack** [comment]
What's the solution? I think we should still show the link even if there is no preview.

**Comment by wa0x6e** [comment]
> What's the solution? I think we should still show the link even if there is no preview.

See updated description

**Comment by Sekhmet** [comment]
> Go to a proposal with an invalid discussion link

Can you link to an example? It's always easier to just click on ready link :D

**Comment by wa0x6e** [comment]
> > Go to a proposal with an invalid discussion link
> 
> Can you link to an example? It's always easier to just click on ready link :D

https://testnet.snapshotx.xyz/#/s-tn:wan-test.eth/proposal/0x04a142d72b0d9b004965be01c3eff9bdad7888ceccf7e053cde129ba92a4ab03

File: apps/ui/src/components/Ui/LinkPreview.vue [hunk 1]
```diff
@@ -27,19 +29,25 @@ debouncedWatch(
...
+      <div class="px-4 py-3 overflow-hidden">
+        <div class="text-skin-link truncate" v-text="preview.meta.title" />
+        <div
+          v-if="preview.meta.description"
+          class="text-[17px] text-skin-text truncate"
+          v-text="preview.meta.description"
+        />
+      </div>
+    </template>
+    <div v-else-if="showDefault" class="p-3 py-2 flex gap-2 items-center w-full">
```
Comment (bonustrack) [inline apps/ui/src/components/Ui/LinkPreview.vue:48]: Can you use same spacing as the preview?
```suggestion
    <div v-else-if="showDefault" class="px-4 py-3 flex gap-2 items-center w-full">
```

**Review by bonustrack — APPROVED** [review]
tACK

**Review by Sekhmet — APPROVED** [review]
tACK

**Pull request merged**